src/rules/selector-attribute-quotes/index.js

```diff
@@ -33,6 +33,8 @@ export default function (expectation) {
       parseSelector(rule.selector, result, rule, selectorTree => {
 
         selectorTree.walkAttributes(attributeNode => {
+          if (!attributeNode.operator) { return }
```

Comment:
You should probably remove the line that was doing this before but once per selector (instead of per node): https://github.com/stylelint/stylelint/blob/0867b8c1075d9e47c28d6280372b2673eda276b0/src/rules/selector-attribute-quotes/index.js#L30